docs/extra-build-config.md

```diff
@@ -77,7 +77,7 @@ To remove (or adjust) this delay set these variables in local.conf:
 ## Set overclocking options
 
 The Raspberry Pi can be overclocked. As of now overclocking up to the "Turbo
-Mode" is officially supported by the raspbery and does not void warranty. Check
+Mode" is officially supported by the raspberry and does not void warranty. Check
```

Comment:
Since Raspberry is a product and registered name, shall it be capitalized and changed to "Raspberry PI"?

Reply:
I think that makes sense.